java/maven/src/org/netbeans/modules/maven/execute/defaultActionMappings.xml

```diff
@@ -260,8 +274,9 @@
         <properties>
             <test>${packageClassName}</test>
             <forkMode>once</forkMode>
-            <maven.surefire.debug>${exec.args}</maven.surefire.debug>
+            <maven.surefire.debug>${exec.vmArgs}</maven.surefire.debug>
             <!-- need to declare exec.args property to engage the StartupExtender infrastructure -->
+            <exec.vmArgs/>
```

Comment:
The changes in this file are very illustrative!

The comment "need to declare exec.args property" is now slightly off...

Reply:
Fixed in af77814